src/components/atoms/VisusalisationsSelector.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { SelectField } from 'evergreen-ui';
+
+type Props = {
+    label: string;
+    options: Record<string, string>;
+    onChange: React.ChangeEventHandler<HTMLSelectElement>;
+};
+
+const VisualisationsSelector: React.FC<Props> = (props) => (
+    <SelectField label={props.label} defaultValue={Object.values(props.options)[0]} onChange={props.onChange}>
```

Comment:
Perhaps make a variable for `Object.values(props.options)[0]` making it easier to see what this value means.

Reply:
Post-discussed deemed not necessary.

Only used once.

Takes first value in map.